### ISSUE TITLE
Lux should help applications maintain accessible, consistent focus indicators throughout the page

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/lux-design-system#250]
### User story

As a keyboard user with low vision, I would like to clearly and consistently see which element on the page has focus at any given time, so that I can efficiently navigate through the page.

### Background

Currently, Lux has a variety of different ways of telling the user that a particular element has focus:

- LuxCards may have a large orange ring around them with a shadow effect
- Several components have a 1px "bleu-de-france" ring around them
- Several components rely on the browser's built-in focus indicator
- Menu items in the LuxMenuBar component turn off the browser's built-in focus indicator, and instead underline the text

### Relevant WCAG success criteria

|Success criterion|Level|Text|
|---|---|---|
|2.4.7 Focus Visible|AA|Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible.|
|2.4.11 Focus Not Obscured (minimum)|AA|When a user interface component receives keyboard focus, the component is not entirely hidden due to author-created content.|
|2.4.12 Focus Not Obscured (enhanced|AAA|When a user interface component receives keyboard focus, no part of the component is hidden by author-created content.|
|2.4.13 Focus Appearance|AAA|When the keyboard [focus indicator](https://www.w3.org/TR/WCAG22/#dfn-focus-indicator) is visible, an area of the focus indicator meets all the following: <ul><li>is at least as large as the area of a 2 CSS pixel thick perimeter of the unfocused component or sub-component, and</li><li>has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.</li></ul>

### Acceptance criteria

- [x] The lux CSS distributed in dist/style.css includes CSS rules that add an enhanced focus indicator to any site that imports the CSS
- [x] The focus indicator is ~2 pixels~ 0.25rem thick
- [x] The focus indicator is a color that has at least 3:1 color contrast ratio to white. Use the orange color for the focus indicator. Lux styleguidist has orange-on-white and orange-on-black
- [x] LuxMenuBar uses the enhanced focus indicator, rather than simply underlining
- [x] When the LuxMenuBar component's theme is "dark", the focus indicator is a color with a 3:1 color contrast ratio to the dark background.
- [x] LuxInputButton uses the enhanced focus indicator
- [x] LuxInputText uses the enhanced focus indicator
 

### Links

- [gov.uk's design system has good documentation about their design decisions on focus indicators](https://design-system.service.gov.uk/get-started/focus-states/)
- [The BBC's design system also has some documentation](https://bbc.github.io/gel/foundations/focus/#recommended-layout)
- [Shopify's design system has a design token/variable for the focus indicator color](https://polaris.shopify.com/tokens/color#color-border-focus)
- [Sara Soueidan has a guide about focus indicators](https://www.sarasoueidan.com/blog/focus-indicators/)

**Comment by vh8154** [comment]
@maxkadel Hi Max, I was looking at the code after the meeting just now and I think that for the submenu toggle piece specifically we could possibly add the focus indicator on [line 425 of the LuxMenuBar file](https://github.com/pulibrary/lux-design-system/blob/main/src/components/LuxMenuBar.vue).

**Comment by christinach** [comment]
@maxkadel will update the Header and Footer
